docs/cray-support.md

```diff
@@ -38,146 +19,134 @@ Thanks to:
+### Architecture 
 
-  * e.g., ``--optarch=sandybridge`` results in ``craype-sandybridge`` being loaded in the build environment used by EasyBuild 
+* the `craype-<target>` module to load must be specified using `--optarch`
+    * e.g., `--optarch=sandybridge` results in `craype-sandybridge` being loaded in the build environment used by EasyBuild 
 
-You can also export this option as a shell variable. Example for ``sandybridge``::
+You can also export this option as a shell variable. Example for `sandybridge`:
 
- $ export EASYBUILD_OPTARCH=sandybridge
+``` shell
+export EASYBUILD_OPTARCH=sandybridge
+```
 
-Metadata for Cray-provided modules
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+### Metadata for Cray-provided modules
 
 * Easybuild provides a sample metadata file in order to use modules provided by Cray:
 
-::
-
- easybuild-framework/etc/cray_external_modules_metadata.cfg
-
+``` shell
+easybuild-framework/etc/cray_external_modules_metadata.cfg
+```
 
 This file is loaded by default and contains enough information to build the easyconfig files shipped with EasyBuild.
 
-If you need to use a customized file, it can be specified using ``--external-modules-metadata``.
-For more details see :ref:`using_external_modules_metadata`.
+If you need to use a customized file, it can be specified using `--external-modules-metadata`.
+For more details see [Metadata for external modules][using_external_modules_metadata].
 
-Major supported/tested applications
------------------------------------
+## Major supported/tested applications
 
 (in alphabetical order)
 
-CP2K
-~~~~
+### CP2K
 
-* http://www.cp2k.org
+* <https://www.cp2k.org>
 * version(s): 2.6.0
 
-::
- 
- $ eb CP2K-2.6.0-CrayGNU-2015.06.eb -dr 
+``` shell
+eb CP2K-2.6.0-CrayGNU-2015.06.eb -dr
+```
 
-GROMACS
-~~~~~~~
+### GROMACS
 
-* http://www.gromacs.org
+* <https://www.gromacs.org>
 * version(s): 4.6.7
 
-::
-
- eb GROMACS-4.6.7-CrayGNU-2015.06-mpi.eb -dr 
+``` shell
+eb GROMACS-4.6.7-CrayGNU-2015.06-mpi.eb -dr
+```
 
-HPL
-~~~
+### HPL
 
-* http://www.netlib.org/benchmark/hpl
+* <https://www.netlib.org/benchmark/hpl>
 * version(s): 2.1
 
-::
-
- eb HPL-2.1-CrayCCE-2015.06.eb -dr
- eb HPL-2.1-CrayGNU-2015.06.eb -dr 
- eb HPL-2.1-CrayIntel-2015.06.eb -dr 
+``` shell
+eb HPL-2.1-CrayCCE-2015.06.eb -dr
+eb HPL-2.1-CrayGNU-2015.06.eb -dr 
+eb HPL-2.1-CrayIntel-2015.06.eb -dr 
+```
 
-Python + numpy/scipy
-~~~~~~~~~~~~~~~~~~~~
+### Python + numpy/scipy
 
-* http://python.org, http://www.numpy.org, http://www.scipy.org
+* <https://python.org>, <https://www.numpy.org>, <https://www.scipy.org>
 * version(s): Python 2.7.9, numpy 1.9.2, scipy 0.15.1
 
-::
-
- eb Python-2.7.9-CrayGNU-2015.06.eb -dr 
- # includes a few python packages (such as mpi4py, numpy and scipy)
+``` shell
+eb Python-2.7.9-CrayGNU-2015.06.eb -dr 
+# includes a few python packages (such as mpi4py, numpy and scipy)
+```
 
-WRF
-~~~
+### WRF
 
-* http://www.wrf-model.org
+* <https://www.wrf-model.org>
```

Comment:
This link doesn't work for me. Maybe this one instead, if it's the same software?
```suggestion
* <https://www2.mmm.ucar.edu/wrf/users/>
```